metrics/metrics.go

```diff
@@ -15,17 +17,25 @@ import (
 
 // Collect - collects metrics
 func Collect(network string, peerMap models.PeerMap) (*models.Metrics, error) {
+	mi := 15
```

Comment:
what does `15` denote here?

Reply:
mi means for metric interval,  usually it's got from metric publish interval from server side.   15 is the default value.

Reply:
shouldn't this be fetched from server config?
why is it hardcoded?

Reply:
Yes, it's fetched from server config.  It's initiating with a default value, in case of any error when fetching from server side.

server := config.GetServer(config.CurrServer)
	if server != nil {
		i, err := strconv.Atoi(server.MetricInterval)
		if err == nil && i > 0 {
			mi = i
		}
	}